astropy/io/fits/file.py

```diff
@@ -83,6 +83,7 @@ class _File(object):
     @deprecated_renamed_argument('clobber', 'overwrite', '2.0')
     def __init__(self, fileobj=None, mode=None, memmap=None, overwrite=False,
                  cache=True):
+        from http.client import HTTPResponse
```

Comment:
Ack, I actually just force-pushed to work around support for Python 2. I could change it to something portable if we wanted to backport, but in my most recent change it just won't work (and won't be tested) for Python 2.